core/root/cluster.go

```diff
@@ -309,9 +309,9 @@ func (c clusterImpl) Update() (string, error) {
 
 	if c.controlPlane.CloudWatchLogging.Enabled && c.controlPlane.CloudWatchLogging.LocalStreaming.Enabled {
 		// Return Journald logs in a separate GoRoutine
-		quit := make(chan bool)
-		defer func() { quit <- true }()
-		go streamJournaldLogs(c, quit)
+		q := make(chan struct{})
```

Comment:
Pro: 1 less byte of memory :smile: https://stackoverflow.com/questions/22770114/any-difference-in-using-an-empty-interface-or-an-empty-struct-as-a-maps-value-i